workshop/exercise-1/README.md

```diff
@@ -12,11 +12,6 @@ You must already have a [cluster created](https://console.bluemix.net/docs/conta
 ibmcloud login [--sso]
 ```
 
-or if you have an api key, use:
-
-```shell
-ibmcloud login --apikey <apikey>
-```
 
```

Comment:
I don't agree with removing this.  In the past, we have been providing the pre-provisioned cluster to users, and using apikey is their way to access the cluster.